gateway/gateway.go

```diff
@@ -1491,6 +1494,38 @@ func (*HandleT) GetWebhookSourceDefName(writeKey string) (name string, ok bool)
 	return
 }
 
+// SaveErrors saves errors to the error db
+func (gateway *HandleT) SaveWebhookFailures(reqs []*model.FailedWebhookPayload) error {
```

Comment:
does this method needs to be exported?

Reply:
Yes. Can't pass `gateway` to webhook.Setup (in gateway.go) without exporting.

Also, there were similar functions in that interface exported.